src/server/server.cpp

```diff
@@ -788,8 +788,10 @@ void server::handle_login(socket_ptr socket, std::shared_ptr<simple_wml::documen
 void server::send_password_request(socket_ptr socket, const std::string& msg,
 								   const std::string& user, const char* error_code, bool force_confirmation)
 {
-	std::string salt = user_handler_->create_salt();
 	std::string pepper = user_handler_->create_pepper(user);
+	std::string salt = (pepper[1] == '2')     // Using crypt_blowfish?
+		? user_handler_->create_secure_salt() // yes, use 32 random Base64 characters, cryptographic-stength, 192 bits entropy
```

Comment:
The lines get too long with these comments.

Please change this ternary to an if-else and place the comments to their own lines.